lib/libutee/sub.mk

```diff
@@ -12,5 +12,32 @@ srcs-y += tee_api_objects.c
 srcs-y += tee_api_operations.c
 srcs-y += tee_api_se.c
 srcs-y += tee_api_panic.c
+srcs-y += utf8.c
+srcs-y += image.c
+srcs-y += image_png.c
+srcs-y += font.c
+
+# Need to locate font.h from generated sources
+incdirs-y += .
+
+gensrcs-y += default_bold
+produce-additional-default_bold = default_bold.h
+produce-default_bold = default_bold.c
+depends-default_bold = scripts/render_font.py \
+		%this-dir%/amble/Amble-Bold.ttf
```

Comment:
Ah, yes of course. Then could we define `$(this-dir)` maybe...? Or `$(src-subdir)` to match `$(out-subdir)` (see my next comment).